from PyQt4 import QtCore, QtGui

try:
    _fromUtf8 = QtCore.QString.fromUtf8
except AttributeError:
    def _fromUtf8(s):
        return s

try:
    _encoding = QtGui.QApplication.UnicodeUTF8
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig, _encoding)
except AttributeError:
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig)

class Ui_Widget(object):
    def setupUi(self, Widget):
        Widget.setObjectName(_fromUtf8("Widget"))
        Widget.resize(926, 506)
        self.groupBox_singlePhotonCounts = QtGui.QGroupBox(Widget)
        self.groupBox_singlePhotonCounts.setGeometry(QtCore.QRect(280, 50, 521, 181))
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.groupBox_singlePhotonCounts.setFont(font)
        self.groupBox_singlePhotonCounts.setObjectName(_fromUtf8("groupBox_singlePhotonCounts"))
        self.label_A = QtGui.QLabel(self.groupBox_singlePhotonCounts)
        self.label_A.setGeometry(QtCore.QRect(50, 30, 31, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_A.setFont(font)
        self.label_A.setObjectName(_fromUtf8("label_A"))
        self.label_B = QtGui.QLabel(self.groupBox_singlePhotonCounts)
        self.label_B.setGeometry(QtCore.QRect(180, 30, 31, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_B.setFont(font)
        self.label_B.setObjectName(_fromUtf8("label_B"))
        self.label_C = QtGui.QLabel(self.groupBox_singlePhotonCounts)
        self.label_C.setGeometry(QtCore.QRect(310, 30, 31, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_C.setFont(font)
        self.label_C.setObjectName(_fromUtf8("label_C"))
        self.label_D = QtGui.QLabel(self.groupBox_singlePhotonCounts)
        self.label_D.setGeometry(QtCore.QRect(440, 30, 31, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.label_D.setFont(font)
        self.label_D.setObjectName(_fromUtf8("label_D"))
        self.textEdit_A = QtGui.QTextEdit(self.groupBox_singlePhotonCounts)
        self.textEdit_A.setGeometry(QtCore.QRect(10, 80, 101, 31))
        self.textEdit_A.setReadOnly(True)
        self.textEdit_A.setObjectName(_fromUtf8("textEdit_A"))
        self.textEdit_B = QtGui.QTextEdit(self.groupBox_singlePhotonCounts)
        self.textEdit_B.setGeometry(QtCore.QRect(140, 80, 101, 31))
        self.textEdit_B.setReadOnly(True)
        self.textEdit_B.setObjectName(_fromUtf8("textEdit_B"))
        self.textEdit_C = QtGui.QTextEdit(self.groupBox_singlePhotonCounts)
        self.textEdit_C.setGeometry(QtCore.QRect(270, 80, 101, 31))
        self.textEdit_C.setReadOnly(True)
        self.textEdit_C.setObjectName(_fromUtf8("textEdit_C"))
        self.textEdit_D = QtGui.QTextEdit(self.groupBox_singlePhotonCounts)
        self.textEdit_D.setGeometry(QtCore.QRect(400, 80, 101, 31))
        self.textEdit_D.setReadOnly(True)
        self.textEdit_D.setObjectName(_fromUtf8("textEdit_D"))
        self.label_warning = QtGui.QLabel(self.groupBox_singlePhotonCounts)
        self.label_warning.setEnabled(False)
        self.label_warning.setGeometry(QtCore.QRect(130, 130, 251, 21))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.label_warning.setFont(font)
        self.label_warning.setObjectName(_fromUtf8("label_warning"))
        self.groupBox_settings = QtGui.QGroupBox(Widget)
        self.groupBox_settings.setGeometry(QtCore.QRect(50, 50, 181, 241))
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.groupBox_settings.setFont(font)
        self.groupBox_settings.setObjectName(_fromUtf8("groupBox_settings"))
        self.pushButton = QtGui.QPushButton(self.groupBox_settings)
        self.pushButton.setGeometry(QtCore.QRect(30, 40, 111, 51))
        font = QtGui.QFont()
        font.setBold(False)
        font.setWeight(50)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName(_fromUtf8("pushButton"))
        self.doubleSpinBox_updatePeriod = QtGui.QDoubleSpinBox(self.groupBox_settings)
        self.doubleSpinBox_updatePeriod.setGeometry(QtCore.QRect(30, 130, 62, 22))
        font = QtGui.QFont()
        font.setBold(False)
        font.setWeight(50)
        self.doubleSpinBox_updatePeriod.setFont(font)
        self.doubleSpinBox_updatePeriod.setDecimals(1)
        self.doubleSpinBox_updatePeriod.setSingleStep(0.1)
        self.doubleSpinBox_updatePeriod.setObjectName(_fromUtf8("doubleSpinBox_updatePeriod"))
        self.doubleSpinBox_coincidenceWindow = QtGui.QDoubleSpinBox(self.groupBox_settings)
        self.doubleSpinBox_coincidenceWindow.setGeometry(QtCore.QRect(30, 190, 62, 22))
        font = QtGui.QFont()
        font.setBold(False)
        font.setWeight(50)
        self.doubleSpinBox_coincidenceWindow.setFont(font)
        self.doubleSpinBox_coincidenceWindow.setDecimals(1)
        self.doubleSpinBox_coincidenceWindow.setSingleStep(0.1)
        self.doubleSpinBox_coincidenceWindow.setObjectName(_fromUtf8("doubleSpinBox_coincidenceWindow"))
        self.label_updatePeriod = QtGui.QLabel(self.groupBox_settings)
        self.label_updatePeriod.setGeometry(QtCore.QRect(30, 110, 91, 16))
        font = QtGui.QFont()
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        self.label_updatePeriod.setFont(font)
        self.label_updatePeriod.setObjectName(_fromUtf8("label_updatePeriod"))
        self.label_coincidenceWindow = QtGui.QLabel(self.groupBox_settings)
        self.label_coincidenceWindow.setGeometry(QtCore.QRect(30, 170, 141, 16))
        font = QtGui.QFont()
        font.setBold(False)
        font.setWeight(50)
        self.label_coincidenceWindow.setFont(font)
        self.label_coincidenceWindow.setObjectName(_fromUtf8("label_coincidenceWindow"))
        self.groupBox_coincidenceCounts = QtGui.QGroupBox(Widget)
        self.groupBox_coincidenceCounts.setGeometry(QtCore.QRect(280, 250, 521, 211))
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.groupBox_coincidenceCounts.setFont(font)
        self.groupBox_coincidenceCounts.setObjectName(_fromUtf8("groupBox_coincidenceCounts"))
        self.label_AB = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_AB.setGeometry(QtCore.QRect(20, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_AB.setFont(font)
        self.label_AB.setObjectName(_fromUtf8("label_AB"))
        self.label_AC = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_AC.setGeometry(QtCore.QRect(60, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_AC.setFont(font)
        self.label_AC.setObjectName(_fromUtf8("label_AC"))
        self.label_AD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_AD.setGeometry(QtCore.QRect(100, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_AD.setFont(font)
        self.label_AD.setObjectName(_fromUtf8("label_AD"))
        self.label_BC = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_BC.setGeometry(QtCore.QRect(140, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_BC.setFont(font)
        self.label_BC.setObjectName(_fromUtf8("label_BC"))
        self.label_BD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_BD.setGeometry(QtCore.QRect(180, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_BD.setFont(font)
        self.label_BD.setObjectName(_fromUtf8("label_BD"))
        self.label_CD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_CD.setGeometry(QtCore.QRect(220, 30, 31, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_CD.setFont(font)
        self.label_CD.setObjectName(_fromUtf8("label_CD"))
        self.label_ABC = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_ABC.setGeometry(QtCore.QRect(260, 30, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_ABC.setFont(font)
        self.label_ABC.setObjectName(_fromUtf8("label_ABC"))
        self.label_BCD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_BCD.setGeometry(QtCore.QRect(310, 30, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_BCD.setFont(font)
        self.label_BCD.setObjectName(_fromUtf8("label_BCD"))
        self.label_ACD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_ACD.setGeometry(QtCore.QRect(360, 30, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_ACD.setFont(font)
        self.label_ACD.setObjectName(_fromUtf8("label_ACD"))
        self.label_ABD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_ABD.setGeometry(QtCore.QRect(410, 30, 41, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_ABD.setFont(font)
        self.label_ABD.setObjectName(_fromUtf8("label_ABD"))
        self.label_ABCD = QtGui.QLabel(self.groupBox_coincidenceCounts)
        self.label_ABCD.setGeometry(QtCore.QRect(460, 30, 51, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_ABCD.setFont(font)
        self.label_ABCD.setObjectName(_fromUtf8("label_ABCD"))

        self.retranslateUi(Widget)
        QtCore.QObject.connect(self.pushButton, QtCore.SIGNAL(_fromUtf8("clicked()")), Widget.close)
        QtCore.QMetaObject.connectSlotsByName(Widget)

    def retranslateUi(self, Widget):
        Widget.setWindowTitle(_translate("Widget", "Widget", None))
        self.groupBox_singlePhotonCounts.setTitle(_translate("Widget", "Single Photon Counts", None))
        self.label_A.setText(_translate("Widget", "A", None))
        self.label_B.setText(_translate("Widget", "B", None))
        self.label_C.setText(_translate("Widget", "C", None))
        self.label_D.setText(_translate("Widget", "D", None))
        self.label_warning.setText(_translate("Widget", "<html><head/><body><p><span style=\" color:#ff0000;\">HIGH COUNT WARNING!</span></p></body></html>", None))
        self.groupBox_settings.setTitle(_translate("Widget", "Settings", None))
        self.pushButton.setText(_translate("Widget", "STOP", None))
        self.label_updatePeriod.setText(_translate("Widget", "Update Period", None))
        self.label_coincidenceWindow.setText(_translate("Widget", "Coincidence Window", None))
        self.groupBox_coincidenceCounts.setTitle(_translate("Widget", "Coincidence Counts", None))
        self.label_AB.setText(_translate("Widget", "AB", None))
        self.label_AC.setText(_translate("Widget", "AC", None))
        self.label_AD.setText(_translate("Widget", "AD", None))
        self.label_BC.setText(_translate("Widget", "BC", None))
        self.label_BD.setText(_translate("Widget", "BD", None))
        self.label_CD.setText(_translate("Widget", "CD", None))
        self.label_ABC.setText(_translate("Widget", "ABC", None))
        self.label_BCD.setText(_translate("Widget", "BCD", None))
        self.label_ACD.setText(_translate("Widget", "ACD", None))
        self.label_ABD.setText(_translate("Widget", "ABD", None))
        self.label_ABCD.setText(_translate("Widget", "ABCD", None))


if __name__ == "__main__":
    import sys
    app = QtGui.QApplication(sys.argv)
    Widget = QtGui.QWidget()
    ui = Ui_Widget()
    ui.setupUi(Widget)
    Widget.show()
    sys.exit(app.exec_())

